wallet/lib/pages/purchase_item/purchase_item_view_model.dart

```diff
@@ -97,6 +97,9 @@ class PurchaseItemViewModel extends ChangeNotifier {
     repository.logPurchaseItem(recipeId: nft.recipeID, recipeName: nft.name, author: nft.creator, purchasePrice: double.parse(nft.price) / kBigIntBase);
   }
 
+
+
+
```

Comment:
Why added these empty lines?

Reply:
As i wrote bottom sheet logic in here so before pushing i removed it so that is why these spaces comes. will fix it in next push